webui/react/src/components/InteractiveTable.tsx

```diff
@@ -298,8 +298,8 @@ const InteractiveTable: InteractiveTable = ({
   const { width: pageWidth } = useResize(containerRef);
   const tableRef = useRef<HTMLDivElement>(null);
   const [ widthData, setWidthData ] = useState({
-    dropLeftStyles: settings?.columnWidths.map(() => ({})),
-    dropRightStyles: settings?.columnWidths.map(() => ({})),
+    dropLeftStyles: settings?.columnWidths.map(() => ({})) ?? [],
+    dropRightStyles: settings?.columnWidths.map(() => ({})) ?? [],
```

Comment:
👍  was this where it was breaking? where columnWidths didn't map to anything causing these styles to be undefined instead of an array?

Reply:
initially it was breaking [here](https://github.com/determined-ai/determined/blob/575cc21f7fb55fdfb0b80f65babb6ddde85106d9/webui/react/src/components/InteractiveTable.tsx#L487) but after an initial fix it was broken here as well. added some defensive coding throughout.